parser/binary-parser.go

```diff
@@ -9,23 +9,34 @@ import (
 	tfjson "github.com/hashicorp/terraform-json"
 )
 
+type CommandExecutor interface {
+	CombinedOutput(name string, args ...string) ([]byte, error)
+}
+
+type RealCommandExecutor struct{}
```

Comment:
Can we name it DefaultCommandExecutor ? which is the common pattern in go community

Reply:
Ok, will do that. Thanks.